code/game/turfs/space/transit.dm

```diff
@@ -1,102 +1,22 @@
 /turf/space/transit
-	var/pushdirection // push things that get caught in the transit tile this direction
+	name = "bluespace"
+	desc = "The blue beyond, a breach into an unknown dimension. Don't lick it."
+	description_info = "Bluespace is a very strange form of pocket dimension, that is largely unpredictable and completely unexplored. While there is speculation about the possibility of celestial bodies existing in Bluespace, it is highly unlikely. Travelling in the Bluespace dimension without a proper gate or Bluespace drive has thus far been proven to be incredibly dangerous, with probes either appearing in unintended locations or never returning at all."
 	plane = 0
+	use_space_appearance = FALSE
+	use_starlight = FALSE
+	light_color = COLOR_CYAN_BLUE
+	light_power = 6
+	light_range = 8
 
 /turf/space/transit/Initialize()
-	update_icon()
-	initialized = TRUE
-	return INITIALIZE_HINT_NORMAL
-
-/turf/space/transit/update_icon()
-	icon_state = ""
-
-	var/dira=""
-	var/i=0
-	switch(pushdirection)
-		if(SOUTH) // North to south
-			dira="ns"
-			i=1+(abs((x^2)-y)%15) // Vary widely across X, but just decrement across Y
-
-		if(NORTH) // South to north  I HAVE NO IDEA HOW THIS WORKS I'M SORRY.  -Probe
-			dira="ns"
-			i=1+(abs((x^2)-y)%15) // Vary widely across X, but just decrement across Y
-
-		if(WEST) // East to west
-			dira="ew"
-			i=1+(((y^2)+x)%15) // Vary widely across Y, but just increment across X
-
-		if(EAST) // West to east
-			dira="ew"
-			i=1+(((y^2)-x)%15) // Vary widely across Y, but just increment across X
-
-
-		/*
-		if(NORTH) // South to north (SPRITES DO NOT EXIST!)
-			dira="sn"
-			i=1+(((x^2)+y)%15) // Vary widely across X, but just increment across Y
-
-		if(EAST) // West to east (SPRITES DO NOT EXIST!)
-			dira="we"
-			i=1+(abs((y^2)-x)%15) // Vary widely across X, but just increment across Y
-		*/
-
-		else
-			icon_state="black"
-	if(icon_state != "black")
-		icon_state = "speedspace_[dira]_[i]"
+	. = ..()
+	icon_state = "bluespace[pick("","3","6","9","12","15")]"
+	update_light()
```

Comment:
this is probably inefficient as fuck because it updates for every single bluespace tile ever. if anyone with a massive brain has a better idea, lemme know

Reply:
Make it a space reskin and use starlight?

Reply:
I'm pretty certain that we ALREADY do update_light whenever a ship jumps. So adding static light sources sounds like the best idea.